WordPress/Classes/Utility/WebKitViewController.swift

```diff
@@ -359,9 +359,18 @@ class WebKitViewController: UIViewController, WebKitAuthenticatable {
 
     /// Sets the width of the web preview
     /// - Parameter width: The width value to set the webView to
-    func setWidth(_ width: CGFloat?) {
+    func setWidth(_ width: CGFloat?, viewSize: CGSize? = nil) {
         if let width = width {
-            widthConstraint?.constant = min(width, view.superview?.frame.width ?? width)
+            var horizontalViewBound: CGFloat
```

Comment:
Swift lets us define this as a constant if it knows we're immediately about to set it:

```suggestion
            let horizontalViewBound: CGFloat
```